frontend/src/components/SettingsModal.jsx

```
@@ -299,467 +299,329 @@ const SettingsModal = ({ isOpen, onClose, onSave, activeModel }) => {
-                            onClick={handleIndex}
-                            disabled={config.folders.length === 0 || indexingStatus.running}
-                            className="w-full py-2.5 rounded-xl bg-primary text-primary-foreground hover:bg-primary/90 disabled:opacity-50 text-sm font-bold shadow-lg shadow-primary/20 transition-all active:scale-[0.98] mt-2"
-                        >
-                            {indexingStatus.running ? 'Indexing...' : 'Rebuild Index'}
-                        </button>
+                            {activeSection === 'providers' && (
+                                <section className="space-y-4">
+                                    <h3 className="text-xs font-bold text-muted-foreground uppercase tracking-widest">API Keys</h3>
+                                    {apiProviders.map(provider => (
+                                        <div key={provider.id} className="space-y-1">
+                                            <label className="text-sm font-medium">{provider.name}</label>
```

Biome (2.4.6)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)